scalding-date/src/main/scala/com/twitter/scalding/Globifier.scala

```diff
@@ -56,27 +56,24 @@ class BaseGlobifier(dur: Duration, val sym: String, pattern: String, tz: TimeZon
     val estr = format(dr.end)
     if (dr.end < dr.start) {
       Nil
+    } else if (child.isEmpty) {
```

Comment:
Whoops.  This reversion is a mistake due to some `git` mistakes on my part.  Let me fix this.